h5p-bildetema/src/components/Bildetema/Bildetema.module.scss

```diff
@@ -1,3 +1,12 @@
+@use "../../abstracts" as a;
+
+.main {
+  display: flex;
+  flex-direction: column;
+  gap: a.$spacing--16;
```

Comment:
```suggestion
@use "../../abstracts" as *;

.main {
  display: flex;
  flex-direction: column;
  gap: $spacing--16;
```

Reply:
For imports other than `sass:*` we don't add aliases, but rather use `@use x as *`, which allows us to use them directly.

Reply:
\+